app/models/imports/uncategorized.rb

```diff
@@ -27,7 +27,7 @@ def process(**)
 
     def create_child!(**kwargs)
       create_import!(
-        status: :pending,
+        status: child_status,
```

Comment:
@littleforest I don't see any other reference to `child_status`: What is this used for? Will it change?

Reply:
nvm just found it: i thought it was :child_status instead of a method call 😁